Anlab.Mvc/Views/TestItems/Index.cshtml

```diff
@@ -81,17 +88,35 @@
     @{ await Html.RenderPartialAsync("_DataTableScriptsPartial"); }
     <script type="text/javascript">
         $(function() {
-            $("#table").dataTable({
+            var table = $("#table").DataTable({
                 "sorting": [[1, "asc"]],
                 "pageLength": 25,
                 "stateSave": true,
                 "stateDuration": 60 * 10,
                 "columnDefs": [
-                    { "orderable": false, "targets": 0 },
+                    { "orderable": false, "targets": [0,3] },
                     { "searchable":false, "targets": 0 }
                 ]
             });
+
+            $('#category').change(function () {
+                table.draw();
+            });
         });
+
+        $.fn.dataTable.ext.search.push(
+            function (settings, data, dataIndex) {
+
+                var category = data[3];
+                var categoryFilter = $('#category').val();
+
+                if (categoryFilter == "All" || category.indexOf(categoryFilter) !== -1) {
```

Comment:
@laholstege just to check, category (data[3]) is a string right?  If so is there a need to do string.indexOf instead of strict equality (i.e. `category === categoryFilter`)?

Reply:
@srkirkland category can be a string like "Soil Water" so it's just to make sure those get included

Reply:
@laholstege ah makes sense.  Ok 🚢